src/get-started/java-binding.md

```diff
@@ -28,25 +28,30 @@ Follow these steps:
 
 - Download and unpack the archive from [GitHub releases][github-releases] into
   some known location. Create an environment variable `EXONUM_HOME` pointing at this
-  location. You can execute the following command to add this variable to your `.profile`
-  and preserve its value for every terminal session.
+  location. You can also add an entry to the `PATH` variable.
 
   ```bash
-  echo 'export EXONUM_HOME=/path/to/your/install/location' >> ~/.profile
+  export EXONUM_HOME=/path/to/your/install/location
```

Comment:
Shan't we make the installation instructions so that they are repeatable, i.e., specify a particular installation directory by default (e.g., `~/bin` with `EXONUM_HOME=~/bin/exonum-java-0.6.0-release/`)?

Reply:
I don't think we should really decide into what directory users should install the application.

Reply:
I am not saying to decide, but give a reasonable default that is reproducible as is.

Reply:
Actually, the reasonable default will lead to some awkward phrases, because we have different directory names for `debug` and `release`. Moreover, I don't really want to depend on the specific version in the docs - it makes easier to forget to update the numbers.

Reply:
We update the numbers with search/replace, don't we? 

I don't think there is a need for any awkward phrases:

"
… to install the latest release to `~/bin` …:
```
wget it from here
cd ~/bin
unzip …-<version>-release.zip
# set all the paths
```

Debug builds are also available on the release page and can be installed similarly.
"

May be (and probably should be, and already is) split into several explained steps.